src/Stripe.Portable/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Newtonsoft.Json" version="8.0.2" targetFramework="portable45-net45+win8" />
```

Comment:
This was necessary, due to ICloneable not being available on Portable platforms, but that version requiring it.